app/javascript/containers/AdminPreviewStatic.js

```diff
@@ -0,0 +1,151 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import Cover from './Cover';
+import HomePage from './HomePage';
+import StaticPage from './StaticPage';
+
+export default function AdminPreviewStatic({ cover, homepage, opencontent }) {
+  const [ state, setState ] = useState({ isHomepage: true });
+  const clickHandler = () => {
+    setState({
+      ...state,
+      isHomepage: !state.isHomepage
+    });
+  };
+  const StaticHeader = () => (
+    <header className="c-header js-header initialized">
+      <div className="mobile-drawer js-mobile-drawer">
+        <div className="wrapper">
+          <ul>
+            <li><a href="/root-level">Root level</a></li>
```

Comment:
I can see a lot of external links here. Shouldn't we “disable” them (removing the `href` attribute)?